### DISCUSSION TITLE
Electron UI

### DISCUSSION THREAD

**Discussion by yo1nkers** [mcmonkeyprojects/SwarmUI#182] [Q&A]
Hi! I've been a long-time user of the UI, and I was just coming with the question that I'm sure a lot of others are wondering

Any ETA where we can maybe expect the Electron UI? I've been eyeing it off for so long and the concept sounds really interesting.

**Comment by mcmonkey4eva** [comment]
hasn't been prioritized as it's such a minor background thing

**Reply by yo1nkers** [reply]
Whenever you get around to looking at it, I'd surely love an update.

I can see the Electron UI being useful for people who prefer to run it without a chromium instance up as it draws a lot of resources. Most people use Chrome or a Chromium based browser as their main browser and for their runtime instances. While pressing "install app" can help (for where it is available), I think seeing it in Electron could be a large bonus. The UI tabs would be a large improvement.

Also, the Chromium UI doesn't like being scaled and always prefers to be in fullscreen. I think an Electron UI could be a fair solution to this,

**Reply by mcmonkey4eva** [reply]
Electron *is* Chromium

**Reply by yo1nkers** [reply]
While that's true It's certainly a lot less resource intensive than the main browser which is generally full of extensions.

I'm sure it's not a priority but if it ever does, I'd love an update
